AzureCalling/Controllers/CallViewController.swift

```diff
@@ -578,13 +585,30 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
         infoHeaderView.updateParticipant(count: callingContext.participantCount)
     }
 
+    private func participantListUpdate() {
+        DispatchQueue.main.async { [weak self] in
+            guard let self = self,
+                  let bottomDrawerViewController = self.bottomDrawerViewController,
+                  bottomDrawerViewController.isViewLoaded else {
+                return
+            }
+
+            let participantListDataSource = ParticipantListDataSource()
+            let participantInfoList = self.getParticipantInfoList()
+            participantListDataSource.createParticipantList(participantInfoList)
+            bottomDrawerViewController.refreshBottomDrawer(dataSource: participantListDataSource)
```

Comment:
I wonder for the class of DataSource type, could it be possible to input the participantInfoList as an initialization parameter/closure (like tokenFetchClosure) input to them? Reason why I think so is, the participantListDataSource is useless until there is participantInfoList/Device data input.

Reply:
Good point. Changed to use closure for the init. Thanks for the suggestion!